LR_Tourist/DA/Services/Repository/IRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DA.Services.Repository
+{
+    public interface IRepository<T>
+        where T : class
+    {
+        IEnumerable<T> GetAll();
+
+        T Get(int id);
+
+        IEnumerable<T> Find(Func<T, Boolean> predicate);
+
+        void Create(T item);
+
+        void Update(T item);
+
+        void Delete(int id);
+
+        void Save();
```

Comment:
Do we really need this method here?
Can you make save operation inside Delete, or Update, or another CRUD operation?
Make life of programmist easier)

Reply:
fixed